miner/Miner.hs

```diff
@@ -336,7 +336,7 @@ cpu cpue tbytes hbytes = do
             n <- Nonce <$> MWC.uniform (envGen e)
             new <- usePowHash (version $ envArgs e) (\p -> mine p n tbytes) hbytes
             terminateWith sch new
-    writeIORef (envStats e) ns
+    writeIORef (envStats e) $ ns * fromIntegral (cores cpue)
```

Comment:
Can we report actual counts instead of just multiplying by the number of cores?

Reply:
Can we do this first as an intermediate step? The true numbers would be part of the reworking I need to do. This here is intended as a quick fix that gives in-the-ballpark numbers.